app/views/bank_accounts/_form.html.slim

```diff
@@ -1,6 +1,6 @@
 .row
   .col-sm-6
-    = simple_form_for(@bank_account) do |f|
+    = simple_form_for @bank_account, remote: @ajax ? true : false do |f|
```

Comment:
@dissident `remote: @ajax` or if you are not sure what ajax is boolean value `remove: !!@ajax`.